packages/css/src/types.ts

```diff
@@ -602,6 +602,16 @@ export interface ThemeStyles {
   [key: string]: ThemeUIStyleObject | undefined
 }
 
+export function getUseRootStyles(theme: Theme = {}): any {
```

Comment:
![image](https://user-images.githubusercontent.com/15332326/105559597-e273d000-5d11-11eb-8416-e9b442b13e22.png)
Pourquoi, Francis? Pourquoi?

Reply:
Haha, look at you leaving comments in my native language 😂

I think I mimicked from another function, I'm not super familiar with typescript.